asv/plugins/virtualenv.py

```diff
@@ -126,17 +127,21 @@ def _setup(self):
         Then, all of the requirements are installed into
         it using `pip install`.
         """
+        import virtualenv
```

Comment:
This plugin actually has a try: except: block which would fail without `virtualenv` being present, so it makes sense to use it.